android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetView.java

```diff
@@ -75,10 +75,10 @@
             new AppInfoModule(reactContext),
             new AudioModeModule(reactContext),
             new ExternalAPIModule(reactContext),
-            new NAT64InfoModule(reactContext),
             new PictureInPictureModule(reactContext),
             new ProximityModule(reactContext),
-            new WiFiStatsModule(reactContext)
+            new WiFiStatsModule(reactContext),
+            new org.jitsi.meet.sdk.net.NAT64AddrInfoModule(reactContext)
```

Comment:
why is only this module with fully qualified package name when the others are not ?